requirements.txt

```diff
@@ -1,29 +1,45 @@
-# before py37
-pip>=20.3; python_version < "3.7"
-async-generator; python_version < "3.7"
-# contextvars; python_version < "3.7"  # uncomment this one for distribution
-dataclasses; python_version < "3.7"
-immutables; python_version < "3.7"
-
-# before py38
-typing-extensions; python_version < "3.8"
-
-# windows only
-pyreadline3; sys_platform == "win32" or sys_platform == "win64"
-
-# core requirements
-click
-fastapi
-httpx
-Jinja2
-lxml
-openpyxl
-pydantic
+aiofiles==0.7.0; python_version >= "3.6"
```

Comment:
this file looks wrong, but only because i'm in the process of cleaning up our requirements. what new packages do you need to run `scriptability`?

```
oyaml
thoughtspot-tml
```

anything else?

Reply:
Maybe.